playground/mocks/config/responseConfig.js

```diff
@@ -250,6 +250,18 @@ const emailVerificationMocks = {
 
 
 // ===== IDX
+const fastpassUnassignedApp = {
+  '/idp/idx/introspect': [
+    'error-400-user-not-assigned',
+  ],
+  '/idp/idx/authenticators/okta-verify/launch': [
+    'error-400-user-not-assigned-2',
```

Comment:
whats' the difference in these 3 data files ?

Reply:
Just the time is different. Adding the mock and flow just to make sure that a new responds is returned and UI is rerendered.

Reply:
if we can keep the same mock file.. would there be any issue ?

Reply:
What's the issue?

Reply:
I mean why create multiple files for `error-400-user-not-assigned`

Reply:
It is to mimic the flow where 1. an app link for an unassigned user is loaded, then 2. the user click OV button or submit username form.

Reply:
but we can use the same mock file right ?

Reply:
If it is the same mock file, we cannot guarantee the UI has the same rerender logic as with different mocks.